### PR TITLE
[Feature] Add "mixup" from Bag of Tricks

### PR DESCRIPTION
**Pull request by ftbabi**
Hi~
This pr aims to add "mixup" trick mentioned in "Bag of Tricks for Image Classification with Convolutional Neural Networks". So far, the "mixup" operation has only one version which is the same as the official implementation.

### CODE REVIEW

**Comment by ycxioooong** [comment]
Unit tests are needed

**Comment by ftbabi** [comment]
Hi, Yu Xiong~
I've added tests for the codes.

File: mmcls/models/losses/cross_entropy_loss.py [hunk 1]
```diff
@@ -75,19 +107,28 @@ class CrossEntropyLoss(nn.Module):
...
+    def __init__(self,
+                 use_sigmoid=False,
+                 use_soft=False,
+                 reduction='mean',
+                 loss_weight=1.0):
         super(CrossEntropyLoss, self).__init__()
         self.use_sigmoid = use_sigmoid
+        self.use_soft = use_soft
         self.reduction = reduction
         self.loss_weight = loss_weight
```
Comment (ycxioooong) [inline mmcls/models/losses/cross_entropy_loss.py:127]: use_soft and use_sigmoid could not be set simultaneously, so we'd better add some assertions here.

File: tests/test_backbones/test_utils.py [hunk 2]
```diff
@@ -114,3 +114,16 @@ def test_inverted_residual():
     x_out = block(x)
     assert block.with_cp
     assert x_out.shape == torch.Size((1, 16, 56, 56))
+
+
+def test_mixup():
```
Comment (ycxioooong) [inline tests/test_backbones/test_utils.py:119]: we may also check the image and gt results after mix_up

**Review by ycxioooong — APPROVED** [review]
LGTM!

**Pull request merged**